shared-module/board/__init__.c

```diff
@@ -167,6 +167,8 @@ void reset_board_busses(void) {
     }
     #endif
     if (!display_using_spi) {
+        // make sure lock is not held over a soft reset
+        common_hal_busio_spi_unlock(&spi_obj);
```

Comment:
I think we want this outside the if because user code could lock and prevent the display from refreshing.

Reply:
I moved the code and tested OK.